examples/pylint/E0302_unexpected_special_method_signature.py

```diff
@@ -0,0 +1,11 @@
+class A:
+	def __init__(self):
```

Comment:
I haven't decided what I want for docstrings for these examples, so for now I will accept this.

@WChrisK seems like your examples use tabs. Please convert indentation to 4 spaces.